### PR TITLE
AP-6503: Changes to show new Log Animation for the new data layer

### PR DESCRIPTION
**Pull request by brucenguyen1**
- Add new menu item with label "Animate Log (new data layer)" below the current menu item for the old log animation.
- Add flag to switch it on/off
- NOTE: this PR must be merged into Core before this PR: https://github.com/apromore/ApromoreEE/pull/1568

### CODE REVIEW

**Review by dreamind — APPROVED** [review]

File: Apromore-Boot/src/main/resources/menus.json [hunk 1]
```diff
@@ -76,6 +76,9 @@
             {
               "id": "org.apromore.plugin.portal.loganimation2.LogAnimationPlugin"
             },
+            {
+              "id": "org.apromore.plugin.portal.loganimation_ee.LogAnimationPlugin"
```
Comment (lahiru-maha) [inline Apromore-Boot/src/main/resources/menus.json]: Is it possible to have a package structure similar to `org.apromore.ee.plugin.portal.loganimation.LogAnimationPlugin` ?

File: Apromore-Core-Components/Apromore-Portal/src/main/resources/default-menus.json [hunk 2]
```diff
@@ -76,6 +76,9 @@
             {
               "id": "org.apromore.plugin.portal.loganimation2.LogAnimationPlugin"
             },
+            {
+              "id": "org.apromore.plugin.portal.loganimation_ee.LogAnimationPlugin"
```
Comment (lahiru-maha) [inline Apromore-Core-Components/Apromore-Portal/src/main/resources/default-menus.json]: Same as the above comment. Would be nice if the package structure could be something like `org.apromore.ee.plugin.portal.loganimation.LogAnimationPlugin`

File: Apromore-Boot/src/main/resources/menus.json (hunk 1 above)
Comment (brucenguyen1) [reply Apromore-Boot/src/main/resources/menus.json]: Done

File: Apromore-Core-Components/Apromore-Portal/src/main/resources/default-menus.json (hunk 2 above)
Comment (brucenguyen1) [reply Apromore-Core-Components/Apromore-Portal/src/main/resources/default-menus.json]: Done

**Review by lahiru-maha — APPROVED** [review]
Looks good. Thank you for addressing the code review comments.

**Pull request merged**